setup.cfg

```diff
@@ -1,10 +1,9 @@
-[nosetests]
-nocapture=1
-cover-package=zope.annotation
-cover-erase=1
-with-doctest=0
-where=src
-
 [aliases]
 dev = develop easy_install zope.annotation[testing]
 docs = easy_install zope.annotation[docs]
+
+[bdist_wheel]
+universal = 1
+
+[zest.releaser]
+with create-wheel = yes
```

Comment:
@jamadden Do you mind creating a feature request at `zest.releaser` for creating wheels if `[bdist_wheel] universal = 1` is set?